lib/core-integration/src/Test/Integration/Framework/TestData.hs

```diff
@@ -263,11 +264,16 @@ errMsg403TxTooBig n = "I had to select " ++ show n ++ " inputs to construct the\
     \ that is too big, and this would consequently be rejected by a core node.\
     \ Try sending a smaller amount."
 
-errMsg404MalformedTxPayload :: String
-errMsg404MalformedTxPayload = "I couldn't decode the payload that seems to be an\
-    \ externally-signed transaction due to: wrongly constructed binary blob. \
-    \Make sure to send a base64-encoded binary blob, in the proper binary \
-    \format of the already-serialized transaction."
+errMsg400MalformedTxPayload :: String
+errMsg400MalformedTxPayload =
+    "I couldn't verify that the payload has the correct binary \
+    \format due to: wrongly constructed binary blob. \
+    \I won't send it to the node. Make sure the blob has the correct binary \
+    \format."
```

Comment:
This message seems to repeat itself a bit. How about:
```hs
errMsg400MalformedTxPayload =
    "I couldn't verify that the payload has the correct binary format. \
    \Therefore I couldn't send it to the node. Please check the format \
    \and try again."
```

Reply:
done

Reply:
More conciseness: 👍 

But:

> Therefore I **couldn't** send it to the node.

I'm not sure the wallet is _unable_ to send the payload to the node. It _could_ skip verifying the payload, and just forward it to the node, right?

There is a small but real possibility that the wallet's notion of a correct transaction gets out of sync with the node's.

With that in mind I think it would be more transparent to say that the wallet _chooses_ not to send the transaction to the node. `won't` is one way to achieve that.

Counterarguments: 
- Could argue that `couldn't` already contains that according-to-myself meaning
- Could argue that this distinction _should_ be hidden from users.